console/src/app/modules/policies/message-texts/message-texts.component.ts

```diff
@@ -596,7 +596,7 @@ export class MessageTextsComponent implements OnInit, OnDestroy {
           return handler((this.service as ManagementService).setCustomDomainClaimedMessageCustomText(this.updateRequest));
         case MESSAGETYPES.PASSWORDLESS:
           return handler(
-            (this.service as ManagementService).getCustomPasswordlessRegistrationMessageText(this.updateRequest),
+            (this.service as ManagementService).setCustomPasswordlessRegistrationMessageCustomText(this.updateRequest),
           );
         case MESSAGETYPES.PASSWORDCHANGE:
           return handler((this.service as ManagementService).getCustomPasswordChangeMessageText(this.updateRequest));
```

Comment:
This is also wrong, but it seems like it involves creating a new GRPC method and all. Not sure if I can invest the time figuring this out rather soon.

Reply:
Thx for the fix :) Do you mean the typing of the updateRequest?

Reply:
Hi, sorry for the long silence. Yes, the GRPC typing for updating the password change texts were missing and I didn't have time to invest further at that point.